ftp-test-utils/src/main/java/org/ballerinalang/stdlib/ftp/testutils/mockServerUtils/MockFTPServer.java

```diff
@@ -102,11 +138,179 @@ public static void initServer(BMap<Object, Object> config) throws BallerinaFTPEx
+        ssl.setKeystoreFile(new File("tests/resources/keystore.jks"));
+        ssl.setKeystorePassword("changeit");
+        factory.setSslConfiguration(ssl.createSslConfiguration());
+        factory.setImplicitSsl(true);
+
+        final PropertiesUserManagerFactory userManagerFactory = new PropertiesUserManagerFactory();
+        final UserManager userManager = userManagerFactory.createUserManager();
+        BaseUser user = new BaseUser();
+        user.setName(username);
+        user.setPassword(password);
+        File dataDirectory = new File("tests/resources/datafiles");
+        user.setHomeDirectory(dataDirectory.getAbsolutePath());
+        List<Authority> authorities = new ArrayList<>();
+        authorities.add(new WritePermission());
+        user.setAuthorities(authorities);
+        userManager.save(user);
+        serverFactory.setUserManager(userManager);
+        factory.setPort(port);
+        serverFactory.addListener("default", factory.createListener());
+        ftpsServer = serverFactory.createServer();
+        ftpsServer.start();
+
+        int i = 0;
+        while ((ftpsServer.isStopped() || ftpsServer.isSuspended()) && i < 10) {
+            try {
+                TimeUnit.MILLISECONDS.sleep(500);
+                i++;
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+                throw new BallerinaFTPException("Error in starting Apache FTPS server");
+            }
+        }
+        if (i < 10) {
+            logger.info("Started Apache FTPS server");
+        } else {
+            logger.info("Could not start Apache FTPS server");
+        }
+    }
+
+    public static void initSftpServer(BMap<Object, Object> config) throws IOException, BallerinaFTPException,
+            UnrecoverableKeyException, CertificateException, NoSuchAlgorithmException, KeyStoreException {
+        int port = FTPUtil.extractPortValue(config.getIntValue(StringUtils.fromString(
+                FTPConstants.ENDPOINT_CONFIG_PORT)));
+        final BMap auth = config.getMapValue(StringUtils.fromString(
+                FTPConstants.ENDPOINT_CONFIG_AUTH));
+        final String username;
+        final String password;
+        if (auth != null) {
+            final BMap basicAuth = auth.getMapValue(StringUtils.fromString(
+                    FTPConstants.ENDPOINT_CONFIG_BASIC_AUTH));
+            if (basicAuth != null) {
+                username = (basicAuth.getStringValue(StringUtils.fromString(FTPConstants.ENDPOINT_CONFIG_USERNAME)))
+                        .getValue();
+                password = (basicAuth.getStringValue(StringUtils.fromString(FTPConstants.ENDPOINT_CONFIG_PASS_KEY)))
+                        .getValue();
+            } else {
+                username = null;
+                password = null;
+            }
+        } else {
+            username = null;
+            password = null;
+        }
```

Comment:
```suggestion
        String username = null;
        String password = null;
        if (auth != null) {
            final BMap basicAuth = auth.getMapValue(StringUtils.fromString(
                    FTPConstants.ENDPOINT_CONFIG_BASIC_AUTH));
            if (basicAuth != null) {
                username = (basicAuth.getStringValue(StringUtils.fromString(FTPConstants.ENDPOINT_CONFIG_USERNAME)))
                        .getValue();
                password = (basicAuth.getStringValue(StringUtils.fromString(FTPConstants.ENDPOINT_CONFIG_PASS_KEY)))
                        .getValue();
            }
        }
```

Reply:
Reduced the lines with null assign with a similar manner.